server/edd/load/tests/files/defaultunit_data.json

```diff
@@ -0,0 +1 @@
+[{"model": "main.defaultunit", "pk": 5, "fields": {"measurement_type": 1, "unit": 1, "protocol": null, "parser": ""}}, {"model": "main.defaultunit", "pk": 6, "fields": {"measurement_type": 2, "unit": 17, "protocol": 8, "parser": "Ambr"}}, {"model": "main.defaultunit", "pk": 7, "fields": {"measurement_type": 4, "unit": 1, "protocol": null, "parser": ""}}, {"model": "main.defaultunit", "pk": 8, "fields": {"measurement_type": 8, "unit": 14, "protocol": 8, "parser": "Ambr"}}, {"model": "main.defaultunit", "pk": 9, "fields": {"measurement_type": 18, "unit": 19, "protocol": 8, "parser": "Ambr"}}, {"model": "main.defaultunit", "pk": 10, "fields": {"measurement_type": 9, "unit": 17, "protocol": 8, "parser": "Ambr"}}, {"model": "main.defaultunit", "pk": 11, "fields": {"measurement_type": 10, "unit": 18, "protocol": 8, "parser": "Ambr"}}, {"model": "main.defaultunit", "pk": 12, "fields": {"measurement_type": 11, "unit": 18, "protocol": 8, "parser": "Ambr"}}, {"model": "main.defaultunit", "pk": 13, "fields": {"measurement_type": 12, "unit": 1, "protocol": 8, "parser": "Ambr"}}, {"model": "main.defaultunit", "pk": 14, "fields": {"measurement_type": 13, "unit": 17, "protocol": 8, "parser": "Ambr"}}, {"model": "main.defaultunit", "pk": 15, "fields": {"measurement_type": 16, "unit": 17, "protocol": 8, "parser": "Ambr"}}, {"model": "main.defaultunit", "pk": 16, "fields": {"measurement_type": 14, "unit": 17, "protocol": 8, "parser": "Ambr"}}, {"model": "main.defaultunit", "pk": 17, "fields": {"measurement_type": 15, "unit": 17, "protocol": 8, "parser": "Ambr"}}, {"model": "main.defaultunit", "pk": 18, "fields": {"measurement_type": 17, "unit": 17, "protocol": 8, "parser": "Ambr"}}, {"model": "main.defaultunit", "pk": 19, "fields": {"measurement_type": 7, "unit": 16, "protocol": 8, "parser": "Ambr"}}, {"model": "main.defaultunit", "pk": 20, "fields": {"measurement_type": 6, "unit": 15, "protocol": null, "parser": ""}}]
```

Comment:
Should use a Factory for generating these in the tests that need them. Using a fixture file will hard-code ID values that may not be correct as the code changes.